### PR TITLE
Add uml diagram for retag, untag, Label commands

### PR DESCRIPTION
**Pull request by li-s**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -204,6 +207,9 @@ rename the `Tag` specified by the unique tag name with a different tag name.
 This command checks the presence of the `Tag` using `java.io.File.exists()`, and that the new tag name is unique, i.e. not present in the `AddressBook`.
```
Comment (anthony6401) [inline docs/DeveloperGuide.md]: I think this is incorrect. We don't use File.exists().

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -141,6 +141,9 @@ stores a `Label`, which is an optional field in `Tag`. A `Label` must only conta
 one word. `Label` allows the user to label their tagged file. The purpose is to keep label short and concise, as it only 
 serves as extra information of a tagged file.
 
+This sequence diagram shows a successful execution of `LabelCommand`.
+![LabelCommandSuccessExecution](images/LabelCommandSequenceDiagram.png)
```
Comment (luo-git) [inline docs/DeveloperGuide.md:145]: Need to `<br>` or newline before the diagram.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -196,13 +199,19 @@ removes the `Tag` specified by the unique tag name from the `AddressBook`.
 
 This command checks the existence of the `Tag` with `model.findFilteredTagList()`, and calls method `model.deleteTag()` to delete it.
 
+This sequence diagram shows a successful execution of `UntagCommand`.
+![UntagCommandSuccessExecution](images/UntagSequenceDiagram.png)
```
Comment (luo-git) [inline docs/DeveloperGuide.md:203]: Same as previous

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -196,13 +199,19 @@ removes the `Tag` specified by the unique tag name from the `AddressBook`.
...
 [RetagCommand](https://github.com/AY2021S1-CS2103T-F12-1/tp/blob/master/src/main/java/seedu/address/logic/commands/RetagCommand.java)
 rename the `Tag` specified by the unique tag name with a different tag name.
 
-This command checks the presence of the `Tag` using `java.io.File.exists()`, and that the new tag name is unique, i.e. not present in the `AddressBook`.
-It then gets the filepath of the `Tag` before safely deleting it. Then, a new `Tag` is created with the filepath, and the new tag name.
+This command checks the presence of the `TagName` using `model.findFilteredTagList()`, and that the new tag name is unique, i.e. not present in the `AddressBook`.
+It then sets the tag with the new `TagName`.
+
+This sequence diagram shows a successful execution of `RetagCommand`.
+![RetagCommandSuccessExecution](images/RetagSequenceDiagram.png)
```
Comment (luo-git) [inline docs/DeveloperGuide.md:214]: Same as before.

File: docs/diagrams/LabelCommandSequenceDiagram.puml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+create CommandResult
+LabelCommand --> CommandResult
+activate CommandResult
+
+CommandResult --> LabelCommand :
+deactivate CommandResult
+
+[<- LabelCommand : result
+deactivate LabelCommand
+
```
Comment (luo-git) [inline docs/diagrams/LabelCommandSequenceDiagram.puml:60]: Add a line `destroy LabelCommand`

File: docs/diagrams/RetagSequenceDiagram.puml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+create CommandResult
+RetagCommand --> CommandResult
+activate CommandResult
+
+CommandResult --> RetagCommand :
+deactivate CommandResult
+
+[<- RetagCommand : result
+deactivate RetagCommand
+
```
Comment (luo-git) [inline docs/diagrams/RetagSequenceDiagram.puml:53]: `destroy RetagCommand`

File: docs/diagrams/UntagSequenceDiagram.puml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+create CommandResult
+UntagCommand --> CommandResult
+activate CommandResult
+
+CommandResult --> UntagCommand :
+deactivate CommandResult
+
+[<- UntagCommand : result
+deactivate UntagCommand
```
Comment (luo-git) [inline docs/diagrams/UntagSequenceDiagram.puml:37]: `destroy UntagCommand`

**Review by luo-git — CHANGES_REQUESTED** [review]
Need some small changes.😜

**Review by luo-git — APPROVED** [review]
Your approval rating has reached 100%.

**Pull request merged**